frontend/src/store/index.js

```diff
@@ -1,29 +1,81 @@
 import Vue from "vue";
 import Vuex from "vuex";
-
-// import example from './module-example'
+import axios from "axios";
 
 Vue.use(Vuex);
 
-/*
- * If not building with SSR mode, you can
- * directly export the Store instantiation;
- *
- * The function below can be async too; either use
- * async/await or return a Promise which resolves
- * with the Store instance.
- */
+// Axios config
+const frontendUrl = "http://127.0.0.1:8080/";
+const backendUrl = "http://127.0.0.1:5000";
```

Comment:
For the urls, does this call the live production backend or the local host backend? I'm not too sure about the difference.

Reply:
Resolved!